scripts/zones/Windurst_Waters_[S]/npcs/Mindala-Andola_CC.lua

```diff
@@ -1,29 +1,101 @@
 -----------------------------------
--- Area: Windurst Waters (S)
---   NPC: Mindala-Andola, C.C.
--- Type: Sigil
--- @zone 94
--- !pos -31.869 -6.009 226.793
```

Comment:
I failed to notice it sooner sorry, but we want to keep the `!pos` lines. juts move the zone id onto the end of it

`-- !pos -31.869 -6.009 226.793 94`

Reply:
`Andola-wtfever`..lol